app/models/prison.rb

```diff
@@ -35,6 +35,10 @@ def last_bookable_date(today = Time.zone.today)
     today + booking_window
   end
 
+  def bookable_date?(requested_date = Time.zone.today)
+    available_slots.each(&:iso8601).include?(requested_date)
```

Comment:
How does this work when `iso8601` returns a string like`2015-12-09T09:00/11:00` and `requested_date` is a `Date`?